test/Microsoft.NET.Build.Tests/GivenThatWeWantToReferenceAProject.cs

```diff
@@ -41,15 +41,13 @@ public enum ReferenceBuildResult
         }
 
         [Theory]
-        [InlineData("netcoreapp1.1", true, "netstandard1.5", true, true, true)]
         [InlineData("netstandard1.2", true, "netstandard1.5", true, false, false)]
         [InlineData("netcoreapp1.1", true, "net45;netstandard1.5", true, true, true)]
         [InlineData("netcoreapp1.1", true, "net45;net46", true, false, false)]
         [InlineData("netcoreapp1.1;net461", true, "netstandard1.4", true, true, true)]
         [InlineData("netcoreapp1.1;net45", true, "netstandard1.4", true, false, false)]
         [InlineData("netcoreapp1.1;net46", true, "net45;netstandard1.6", true, true, true)]
         [InlineData("netcoreapp1.1;net45", true, "net46;netstandard1.6", true, false, false)]
-        [InlineData("v4.6.1", false, "netstandard1.4", true, true, true)]
```

Comment:
The unit tests in 'GivenThatWeWantToReferenceAProject' can be broken down into 4 categories:
1. negative single-target unit tests
2. negative multi-target unit tests
3. positive single-target unit tests
4. positive multi-target unit tests

The removed two are in category 3. 

Although the 'v4.6.1/netstandard1.4' is not 'exactly' redundant as the new unit test uses the SDK csproj construction where the removed unit test used the classical method.